src/components/about/About.js

```diff
@@ -37,10 +36,10 @@ const About = () => {
                 PB2020/police-brutality GitHub Repository
               </a>
               {'. '}
-              This repository accumulates and contextualizes crowdsourced
+              This repository accumulates and contextualize crowdsourced
               evidence of police use of force from various sources. Our second
               resource is Twitter. We continually search Twitter for evidence of
-              police use of force with a twitter bot using a technique called
+              police use of force with a Twitter bot using a technique called
               Natural Language Processing.
```

Comment:
Consider adding that the specific NLP model used here is a tailored version of BERT. I also suggest including a link to some information about BERT: https://en.wikipedia.org/wiki/BERT_(language_model)

Reply:
I agree, ill include BERT.